README.md

```diff
@@ -1,6 +1,8 @@
 # Kedge - Concise Application Definition for Kubernetes
 
-[![Build Status Widget]][Build Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack]
+[![Build Status Widget]][Build Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack] 
+
+[![Download Widget Linux]][Bintray Latest Linux] [![Download Widget macOS]][Bintray Latest macOS] [![Download Widget Windows]][Bintray Latest Windows]
```

Comment:
So it seems a bit odd that the widgets at the top have no indication if they're from master or not, or whether it's a tagged release. Maybe a small comment saying

__Latest master binaries:__

What's your opinion?